os/dbuild.sh

```diff
@@ -247,6 +250,103 @@ function SELECT_CONFIG()
 	CONFIGURE ${BOARD}/${CONFIG} || exit 1
 }
 
+function SELECT_PORT()
```

Comment:
this function calls SELECT_DL. 
how about moving this function below the SELECT_DL?